plugins/rst.lisp

```diff
@@ -4,20 +4,37 @@
 (defpackage :coleslaw-rst
   (:use :cl)
   (:import-from :coleslaw #:render-text)
-  (:import-from :docutils #:read-rst #:write-part #:register-settings-spec
-                          #:visit-node #:write-document)
-  (:import-from :docutils.writer.html #:html-writer #:write-part)
+  (:import-from :docutils
+                #:document
+                #:read-rst
+                #:register-settings-spec
+                #:visit-node
+                #:write-document)
+  (:import-from #:docutils.writer.html
+                #:html-writer
+                #:parts
+                #:body-pre-docinfo
+                #:body
+                 )
   (:export #:enable))
 
 (in-package :coleslaw-rst)
 
+
+(defmethod visit-node :after ((writer html-writer) (document document))
+  "This method removes unnecessary HTML elements, such as html, head, body
+and make docutils output only html fragment with document itself."
+  
+  (setf (slot-value writer 'parts) '(body-pre-docinfo
```

Comment:
Instead of adding an after method wouldn't passing a `:parts` initarg `'(body-pre-doc-info body)` to the writer in `RENDER-TEXT` accomplish our goal? ie

```lisp
(let ((writer (make-instance 'html-writer :parts '(body-predocoinfo body)))
          ..)
      ...)
```